testsuite/tests/singlecluster/gateway/reconciliation/change_targetref/conftest.py

```diff
@@ -104,5 +121,6 @@ def _apply_target_ref(apiobj):
 
         policy.modify_and_apply(_apply_target_ref)
         policy.wait_for_ready()
+        time.sleep(5)  # Extra wait to avoid inconsistent DNS issues, wait_for_ready isn't always enough
```

Comment:
If it is needed and it gets merged, please add the occurance to https://github.com/Kuadrant/testsuite/issues/660
No need to address this now